config/docker-compose-worker-distributed.properties

```diff
@@ -0,0 +1,84 @@
+
+# Bootstrap Kafka servers. If multiple servers are specified, they should be comma-separated.
+bootstrap.servers=kafka:9092
+connector.class = "com.ably.kafka.connect.ChannelSinkConnector"
+# The group ID is a unique identifier for the set of workers that form a single Kafka Connect
+# cluster
+group.id=connect-cluster
+
+# The converters specify the format of data in Kafka and how to translate it into Connect data.
+# Every Connect user will need to configure these based on the format they want their data in
+# when loaded from or stored into Kafka
+key.converter=org.apache.kafka.connect.converters.ByteArrayConverter
+key.converter.schema.registry.url=http://confluent:8081
+value.converter=org.apache.kafka.connect.converters.ByteArrayConverter
+value.converter.schema.registry.url=http://confluent:8081
+
+
+# Internal Storage Topics.
+#
+# Kafka Connect distributed workers store the connector and task configurations, connector offsets,
+# and connector statuses in three internal topics. These topics MUST be compacted.
+# When the Kafka Connect distributed worker starts, it will check for these topics and attempt to create them
+# as compacted topics if they don't yet exist, using the topic name, replication factor, and number of partitions
+# as specified in these properties, and other topic-specific settings inherited from your brokers'
+# auto-creation settings. If you need more control over these other topic-specific settings, you may want to
+# manually create these topics before starting Kafka Connect distributed workers.
+#
+# The following properties set the names of these three internal topics for storing configs, offsets, and status.
+config.storage.topic=connect-configs
+offset.storage.topic=connect-offsets
+status.storage.topic=connect-statuses
+
+# The following properties set the replication factor for the three internal topics, defaulting to 3 for each
+# and therefore requiring a minimum of 3 brokers in the cluster. Since we want the examples to run with
+# only a single broker, we set the replication factor here to just 1. That's okay for the examples, but
+# ALWAYS use a replication factor of AT LEAST 3 for production environments to reduce the risk of
+# losing connector offsets, configurations, and status.
+config.storage.replication.factor=1
+offset.storage.replication.factor=1
+status.storage.replication.factor=1
+
+
+# The config storage topic must have a single partition, and this cannot be changed via properties.
+# Offsets for all connectors and tasks are written quite frequently and therefore the offset topic
+# should be highly partitioned; by default it is created with 25 partitions, but adjust accordingly
+# with the number of connector tasks deployed to a distributed worker cluster. Kafka Connect records
+# the status less frequently, and so by default the topic is created with 5 partitions.
+#offset.storage.partitions=25
+#status.storage.partitions=5
```

Comment:
In this properties file there are a lot of commented lines and information about stuff that we are not using, I know that they might be helpful in the future but are we okay with committing them to the repository? :thinking:

Reply:
These comments were part of example configuration file. In my opinion it's good to keep those commented code on configuration files to help customers about what they can use. After all, these configuration files are just examples and will eventually be overriden (or not even used) by customers.